src/libraries/System.Net.WebProxy/src/System/Net/WebProxy.cs

```diff
@@ -158,6 +159,7 @@ private bool IsLocal(Uri host)
 
             string hostString = host.Host;
 
+#pragma warning disable CA1416 // Validate platform compatibility, TODO: its need browser specific implementation
```

Comment:
Does anything from this assembly meaningfully work on browser?  I'd have thought the whole assembly would be marked unsupported.

Reply:
I don't know the details, created an issue for handling this https://github.com/dotnet/runtime/issues/43751